client/src/GamemasterInGame/PlayerList.tsx

```diff
@@ -217,7 +217,13 @@ export function NightPlayerList() {
   );
 }
 
-export function IngamePlayerList() {
+interface IngamePlayerListProps {
+  hideGmNotes?: boolean;
```

Comment:
I don't think we want this, we should just make a spyview note type instead like we do when we show players other notes.